docs/src/1_Mechanics/04_Powers.yaml

```
@@ -918,4 +918,202 @@ Amplify Ally:
+    Description: Sometimes, moving in the shadows is easiest when no one knows where you're going.
+    XP: 2
+    PP: 2
+    Prereq:
+        Role: Martial
+        Level: 3
+        Skill: Stealth > 1
+        Power: Hidden Strike
+    Range: 10
+    Target: 1
+    Save:
+        Trigger: If the space is occupied, the creature occupying that space makes a 
+        Type: STR
+        Fail: they are pushed out of the way
+        Succeed: instead, choose a space within 1 space of the creature to appear. 
+
+Summon Creature:
+    Type: Major
+    Category:
+        - Companion
+        - Summoner
+    Mechanic: Summon a creature to assist you. This creature uses the 'Summoned Creature' stat block and is treated as a Companion for the duration. You choose the creature's form. 
+    Description: It's good to have a friend, even when you have to summon them from somewhere else. Who ever said no to a little extra help?
+    XP: 2
+    PP: 2
+    Prereq:
+        Role: Caster
+        Level: 2
+        Skill: Knowledge > 0
+    Duration: 1 hour
+
+Phase Swap:
+    Type: Minor
+    Category:
+        - Companion
+        - Movement
+        - Summoner
+    Mechanic: Use half your speed to swap spaces with a Summoned Creature 
+    Description: It can often be better to change places with a friendly creature. You'd prefer to be there, or they'd prefer to be here.
+    XP: 2
+    PP: 1
+    Prereq:
+        Role: Caster
+        Level: 3
+        Skill: Knowledge > 1
+        Power: Summon Creature
+    Range: 10
+
+Life Link: 
+    Type: Passive
+    Category: 
+        - Companion
+        - Summoner   
+    Mechanic: While you have at least one summoned creature active, when you take damage you can choose to apply that damage to your summoned creature instead.
+    Description: Channeling energy between friends can sometimes be mutually beneficial. Other times, it's simply beneficial.
+    XP: 2
+    Prereq:
+        Role: Caster
+        Level: 3
+        Skill: Knowledge > 1
+        Power: Summon Creature
+
+Create Potion:
+    Type: Major
+    Category:
+        - Recovery
+        - Alchemist
+    Mechanic: Create a Potion of Healing. After successfully completing a Quick Rest, you can exchange one Rest Card for a Potion of Healing without expending Power Points. All Potions created last until the end of your next Full Rest.
+    Description: It's always good to have some spare healing on hand. 
+    XP: 2
+    PP: 1
+    Prereq:
+        Role: Support
+        Level: 2
+        Skill: Craft > 0
+
+Create Poison:
+    Type: Major
+    Category:
+        - Combat
+        - Alchemist
+    Mechanic: Create a Poison Bottle. Apply to a Physical Weapon to force a DR 2 VIT Save, on a failure they take 1 damage bypassing AP. All Poison Bottles created last until the end of your next Full Rest.
```

Physical weapon is capitalized here. What does it mean? Is it in the glossary? Similar to above, I don't think this sets an upper limit

---

Not sure if we need to implement Physical Weapon as full addition/change, but not sure how else to specify that this can go on a corporeal weapon (sword, axe, arrow) but couldn't be added to a Mystic Attack. 

Maybe the following? 

```suggestion
    Mechanic: Create a Poison Bottle. Poison Bottle can be applied to a weapon. For the duration, all Weapon Attacks force a DR 2 VIT Save, on a failure they take 1 damage bypassing AP. All Poison Bottles created last until the end of your next Full Rest.
```

Would could use your help adjusting the language as needed and maybe even implementing a separate Power that has the save language, so this mechanic could just specify making a poison bottle and applying the effects grants a weapon a separate Power.

---

Sure, this works. But let's add `1 minute` to the description if we don't plan to use it anywhere in the codebase

---

Added in latest commit.